src/main/java/org/apache/solr/benchmarks/QueryGenerator.java

```diff
@@ -23,52 +24,57 @@
 
 public class QueryGenerator {
     final QueryBenchmark queryBenchmark;
-    List<String> queries = new ArrayList<>();
+    // = new ArrayList<>();
+    List<SolrBenchQuery> sbQueries;
     Random random;
     AtomicLong counter = new AtomicLong();
     final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd");
+    List<String> queryStrings = new ArrayList<>();
```

Comment:
Minor nit, can probably remove this field as it's only used locally in the ctor ?